insights/tests/test_syslog.py

```diff
@@ -11,6 +11,9 @@
 Apr 22 10:37:32 boy-bona crontab[28951]: (root) LIST (root)
 Apr 22 10:40:01 boy-bona CROND[30677]: (root) CMD (/usr/lib64/sa/sa1 -S DISK 1 1)
 Apr 22 10:41:13 boy-bona crontab[32515]: (root) LIST (root)
+Apr 29 11:33:36 kvmr7u5 ehtest: crontab[12345]: {
+April 29 11:33:36 kvmr7u5 ehtest: crontab[12345]: {
```

Comment:
The format of this 2 lines is different from others (one more`ehtest:` appeared), if you confirmed it's a valid line, I think the original logic of the `_parse_line` cannot handle it correctly. It's a bug and needs to be fixed.

Would you please to take a double confirm? And let me know the result, Thanks a lot.

Reply:
Actually the format is not changed, because the `ehtest:` is still the hostname, the `crontab[12345]: {` is the message body which comes from other system. The parsing method is still correct.